### ISSUE TITLE
Butterfly has wrong genesis in v1.15.X releases

### ISSUE THREAD

**Issue by travisperson** [filecoin-project/lotus#8647]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [X] I did not make any code changes to lotus.

### Lotus component

- [ ] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [X] Other

### Lotus Version

```text
v1.15.X
```

### Describe the Bug

A genesis committed from January 7th was some how released into v1.15.0

The genesis is from a commit a98ca86a4540cf71c6a6b4a14a94aa0b9b12bafd on Jan 7th

### Logging Information

```text
v1.12.0	8873e3f708621cb2e59913b1a270a6075c680272c6bb181caef78e8cf1533bd2	755b44aa1dfdcaf4c1d9c550e356bdaddb3990d4bad84d4fe9bafe420c66e188
v1.13.0	8873e3f708621cb2e59913b1a270a6075c680272c6bb181caef78e8cf1533bd2	755b44aa1dfdcaf4c1d9c550e356bdaddb3990d4bad84d4fe9bafe420c66e188
v1.13.1	8873e3f708621cb2e59913b1a270a6075c680272c6bb181caef78e8cf1533bd2	755b44aa1dfdcaf4c1d9c550e356bdaddb3990d4bad84d4fe9bafe420c66e188
v1.13.2	8873e3f708621cb2e59913b1a270a6075c680272c6bb181caef78e8cf1533bd2	755b44aa1dfdcaf4c1d9c550e356bdaddb3990d4bad84d4fe9bafe420c66e188
v1.14.0	7c1118307657229e50438495d099d1182d3b913c5ea5a23e1cca948fb744fb0f	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
v1.14.1	7c1118307657229e50438495d099d1182d3b913c5ea5a23e1cca948fb744fb0f	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
v1.14.2	7c1118307657229e50438495d099d1182d3b913c5ea5a23e1cca948fb744fb0f	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
v1.14.3	7c1118307657229e50438495d099d1182d3b913c5ea5a23e1cca948fb744fb0f	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
v1.14.4	7c1118307657229e50438495d099d1182d3b913c5ea5a23e1cca948fb744fb0f	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
v1.15.0	5d843d27da587fdde0194a25f5274b32988a86e454bdc4838d09f961339222aa	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
v1.15.1	5d843d27da587fdde0194a25f5274b32988a86e454bdc4838d09f961339222aa	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
v1.15.2	5d843d27da587fdde0194a25f5274b32988a86e454bdc4838d09f961339222aa	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
release/v1.15.3	5d843d27da587fdde0194a25f5274b32988a86e454bdc4838d09f961339222aa	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
master	5d843d27da587fdde0194a25f5274b32988a86e454bdc4838d09f961339222aa	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
```

### Repo Steps

```
refs=(
  v1.12.0
  v1.13.0
  v1.13.1
  v1.13.2
  v1.14.0
  v1.14.1
  v1.14.2
  v1.14.3
  v1.14.4
  v1.15.0
  v1.15.1
  v1.15.2
  release/v1.15.3
  master
)

digest() {
  case "$(uname -s)" in
    Linux)
      sha256sum $1
      ;;
    Darwin)
      shasum -a 256 $1
      ;;
  esac
}

for r in "${refs[@]}"; do
  git checkout $r > /dev/null 2>&1
  hg=$(digest ./build/genesis/butterflynet.car  | awk '{print $1}')
  hb=$(digest ./build/bootstrap/butterflynet.pi | awk '{print $1}')
  echo -e "$r\t$hg\t$hb"
done
```

**Comment by travisperson** [comment]
I believe this is a bit of a chicken & egg issue. When we reset the butterfly network it generally happens prior to a release being cut. In this case, we reset the network using v1.14.0-rc5, https://github.com/filecoin-project/lotus/pull/8032 (https://github.com/filecoin-project/lotus/pull/8032/commits/9b68174231737a023344adc76ddf243e925932fc).

When master merged into v1.15.0, we overwrote this change.

```
v1.14.0-rc1	5d843d27da587fdde0194a25f5274b32988a86e454bdc4838d09f961339222aa	c24f629346ec27cf29edc9211d448dd5efe89bad5c8709795f9486ed45263ed9
v1.14.0-rc4	5d843d27da587fdde0194a25f5274b32988a86e454bdc4838d09f961339222aa	c24f629346ec27cf29edc9211d448dd5efe89bad5c8709795f9486ed45263ed9
v1.14.0-rc5	5d843d27da587fdde0194a25f5274b32988a86e454bdc4838d09f961339222aa	c24f629346ec27cf29edc9211d448dd5efe89bad5c8709795f9486ed45263ed9
v1.14.0-rc6	7c1118307657229e50438495d099d1182d3b913c5ea5a23e1cca948fb744fb0f	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
v1.14.0-rc7	7c1118307657229e50438495d099d1182d3b913c5ea5a23e1cca948fb744fb0f	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
v1.14.0	7c1118307657229e50438495d099d1182d3b913c5ea5a23e1cca948fb744fb0f	9518bc53c5575cc33427e3b8b71b8ac7a89a2c8779b4bb6585f3288eaa66bba8
```

**Comment by travisperson** [comment]
Some further investigate into this issue v1.14.0 was merged back into master. This should have updated master to use the `7c1118...44fb0f` genesis. However, it doesn't appear to be a file in PR https://github.com/filecoin-project/lotus/pull/8053.

**Comment by jennijuju** [comment]
closed by #8708
